### DISCUSSION TITLE
Move off of SqlLocalDB for ARM support

### DISCUSSION THREAD

**Discussion by jbogard** [LuckyPennySoftware/AutoMapper#4451] [Ideas]
SQL Local DB isn't supported on ARM (see https://github.com/dotnet/SqlClient/issues/1441), which means I can't run AutoMapper tests on my Mac. I think the way forward would be something like [TestContainers](https://testcontainers.com).

Would probably not be a small change in our integration tests. But as-is, I can't really develop on AutoMapper.

**Comment by lbargaoanu** [comment]
So we would run SQL in a container. It would be nice to know how much slower that would be.

**Comment by lbargaoanu** [comment]
Another option here is Sqlite. While it has its own peculiarities, it's trivial to run our tests with it. Maybe we're lucky and won't hit corner cases in the future. If you want to try it, I can push smth in my branch.
